utils/build_swift/driver_arguments.py

```diff
@@ -271,6 +271,12 @@ def create_argument_parser():
              "library for. %(default)s by default.",
         action=arguments.action.concat, type=arguments.type.shell_split,
         default=None)
+    targets_group.add_argument(
+        "--extra-stdlib-deployment-targets",
+        help="list of additional targets to cross-compile the Swift standard "
+             "library for in addition to default",
+        action=arguments.action.concat, type=arguments.type.shell_split,
+        default=None)
```

Comment:
I added this arg so that you can append to the default list instead of replacing what would be there normally for the current platform. For Darwin, it is macos/ios/watch/tv/etc SDKs for each supported arch (which is a very long list to recreate). For our build I just want to append "fuchsia-x86_64,fuchsia-aarch64" to whatever would be default for the platform.